electron/index.html

```diff
@@ -29,7 +29,8 @@ <h3 class="zoom-panel"><span class="zoom-out zoom glow-on-hover">-</span> / <spa
       <div id="tracker-body">
         <!-- START Game View -->
         <div id="game-deck-list" rv-show="game_in_progress">
-          <h3 class="win-loss" rv-show="showWinLossCounter">Wins: { winCounter } &nbsp;&mdash;&nbsp; Losses: { lossCounter }</h3>
+          <h3 class="win-loss" rv-show="showTotalWinLossCounter">Total - Wins: { totalWinCounter } &nbsp;&mdash;&nbsp; Losses: { totalLossCounter }</h3>
+          <h3 class="win-loss" rv-show="showDeckWinLossCounter">Deck - Wins: { deckWinCounter } &nbsp;&mdash;&nbsp; Losses: { deckLossCounter }</h3>
```

Comment:
Too long. Leads to word wrapping messing up the formatting.

Reply:
After playing with the solution, I actually disagree with myself here. Since deck win-loss doesn't persist while you're clicking around, it makes sense to just say "this is the win/loss for the deck you're playing right now."